concise-mid-tag.cddl

```diff
@@ -86,10 +80,10 @@ $device-id-type-choice /= tagged-uuid-type
 
 instance-claim-map = {
   ? comid.instance-element-name => element-name-map
-  $instance-value-group-choice
+  $$instance-value-group-choice
 }
 
-$instance-value-group-choice = (
+$$instance-value-group-choice //= (
```

Comment:
Minor question: Is the double forward slash // still needed on line 88-90

Reply:
it's not strictly needed but it looks like it's good CDDL style